app/src/main/res/values/strings.xml

```diff
@@ -97,9 +97,10 @@
 
     <!-- Drone -->
     <string name="drone_mission_success">The mission was successfully dispatched to the drone</string>
-    <string name="drone_mission_error">An error occurred while dispatching missing to the drone</string>
+    <string name="drone_mission_error">An error occurred while dispatching missing to the drone : %s</string>
     <string name="drone_pause_success">Mission paused</string>
     <string name="drone_pause_error">Mission pause failed, try again later</string>
     <string name="drone_return_error">Could not launch mission to let the drone come back</string>
     <string name="drone_return_success">The drone is coming back</string>
```

Comment:
The resource `R.string.drone_return_success` appears to be unused. You could remove it.